submissions/bilkisuTace/database/tranportation.sql

```diff
@@ -0,0 +1,45 @@
+-- ----------------------------
+-- Table structure for `passengers`
+-- ----------------------------
+CREATE TABLE `passengers` (
+  `id` int(11) NOT NULL AUTO_INCREMENT,
+  `full_name` varchar(50) NOT NULL,
+  `sex` varchar(20) NOT NULL,
+  `age` int(10) DEFAULT NULL,
```

Comment:
Kindly use `VARCHAR` instead of `INT` for `age`. Also, remove the `DEFAULT` there